ocspd/core/certmodel.py

```diff
@@ -103,6 +103,11 @@ def recycle_staple(self, minimum_validity):
             )
             return False
 
+        # from haproxy docs: [...] The content of this file is optional [...]
+        if len(staple) == 0:
```

Comment:
erm sorry. shouldn't use the webeditor…
lines are meant to go to 115 just before the `staple = OCSPResponseParser(staple)`. will fix this